src/serializers/PnmlSerializer.js

```diff
@@ -42,6 +51,45 @@ export default class PnmlSerializer {
         const classifierElement = doc.createElement(type);
         classifierElement.setAttribute('id', element.id);
 
+        if (element.labels) {
+            element.labels.forEach((label) => {
+                const textElement = doc.createElement('text');
+                const textNode = doc.createTextNode(label.text);
+                textElement.appendChild(textNode);
+
+                const graphicsElement = doc.createElement('graphics');
+                const offsetElement = doc.createElement('offset');
+
+                // Calculate PNML offsets (relative to center of element)
```

Comment:
So you basicly move the figure, so the center becomes the top-left corner, right?

Reply:
So that the top-left corner becomes the center I would say but right :laughing: